codegen/templates/plugin/handler_models.tmpl

```diff
@@ -22,23 +24,27 @@ type {{.Name}}Service interface {
 }
 
 func (p *Plugin) handle{{.Name}}List(ctx context.Context, req router.JSONRequest) (router.JSONResponse, error) {
+	ctx, span := tracing.DefaultTracer().Start(ctx, "{{.MachineName}}-list")
+	defer span.End()
```

Comment:
Yeah, I started writing a tracing middleware to also include more info in span attributes, then basically realized I wasn't sure what the span should be, and that _also_ grafana includes a span for the actual CallResource call that gets sent to the plugin, so it seemed like redundant info. I think it may still be useful to have an optional additional tracing middleware, perhaps in a subsequent PR.